integration_tests/configs/ibc.jsonnet

```diff
@@ -128,9 +131,13 @@ config {
           },
         },
         gas_price: {
-          price: 10000000000000,
+          price: 10000000000000000,
```

Comment:
btw why does it set extremely high? it takes the minimum by default?

Reply:
it's to emphasis that the `max_priority_price` take effects.